modules/nf-core/unifire/main.nf

```diff
@@ -0,0 +1,57 @@
+process UNIFIRE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    container "dockerhub.ebi.ac.uk/uniprot-public/unifire:2025.1" // TODO: Update once Bioconda is available
+    containerOptions {
+        if (workflow.containerEngine in ['singularity', 'apptainer']) {
+            return "--bind unifire:/volume"
+        } else {
+            return "-v ./unifire:/volume"
+        }
+    }
```

Comment:
I'd love it if we could remove this and add this https://gitlab.ebi.ac.uk/uniprot-public/unifire/-/blob/master/docker/scripts/unifire-workflow.sh script to the template or script section instead